packages/wearable_rotary/tizen/src/wearable_rotary_plugin.cc

```diff
@@ -0,0 +1,85 @@
+#include "wearable_rotary_plugin.h"
+
+#include <efl-extension/wearable/circle/efl_extension_rotary.h>
+#include <flutter/encodable_value.h>
+#include <flutter/event_channel.h>
+#include <flutter/event_sink.h>
+#include <flutter/event_stream_handler_functions.h>
+#include <flutter/plugin_registrar.h>
+#include <flutter/standard_method_codec.h>
+
+#include <memory>
+
+#include "log.h"
+
+#define ROTARY_CHANNEL_NAME "flutter.wearable_rotary.channel"
+
+class WearableRotaryPlugin : public flutter::Plugin {
+ public:
+  static void RegisterWithRegistrar(flutter::PluginRegistrar *registrar) {
+    LOG_DEBUG("RegisterWithRegistrar for WearableRotaryPlugin");
+    auto plugin = std::make_unique<WearableRotaryPlugin>();
+    plugin->setupEventChannels(registrar);
+    registrar->AddPlugin(std::move(plugin));
+  }
+
+  WearableRotaryPlugin() {}
+
+  virtual ~WearableRotaryPlugin() {}
+
+ private:
+  void setupEventChannels(flutter::PluginRegistrar *registrar) {
```

Comment:
There's only a single channel. Also the function name should start with a capital letter.

```suggestion
  void SetupEventChannel(flutter::PluginRegistrar *registrar) {
```

https://google.github.io/styleguide/cppguide.html#Function_Names